ui/core/sim_manager.ts

```diff
@@ -40,13 +40,32 @@ function newSignals(): SimSignals {
 	}
 }
 
+export function generateRequestId(type: RequestTypes) {
+	let id: string;
+	switch(type) {
+		case RequestTypes.RaidSim:
+			id = 'raidsim';
+			break;
+		case RequestTypes.StatWeights:
+			id = 'statsim';
+			break;
+		case RequestTypes.BulkSim:
+			id = 'bulksim';
+			break;
+		default:
+			id = '???????';
```

Comment:
Is this intentional?

Reply:
I removed the default case by excluding the "ALL" type in the param.